x-pack/plugins/security_solution/public/resolver/view/panels/node_list.tsx

```diff
@@ -53,6 +54,31 @@ interface ProcessTableView {
   href: string | undefined;
 }
 
+const StyledButtonTextContainer = styled.div`
+  align-items: center;
+  display: flex;
+  flex-direction: row;
+`;
+
+const StyledProcessLabel = styled.div`
+  display: inline-block;
+  flex: 3;
+  min-width: 0;
+
+  & .origin-label {
+    color: ${(props) => props.color};
+    font-size: 10.5px;
+    font-weight: 700;
+  }
+
+  & > div {
```

Comment:
Would you use this pattern here as well? https://styled-components.com/docs/advanced#referring-to-other-components